packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk

```diff
@@ -114,10 +116,34 @@
 window.MOJFrontend.initAll();
 
 if (typeof MOJFrontend.MultiFileUpload !== 'undefined') {
+  var temporaryDocInput = document.getElementById('tempDocs');
+
+  function onFileUpload(context, file, response) {
+    var upload = {
+      id: response.file.id,
```

Comment:
worth checking for the existence of `response.file.id` first?

Reason I ask is that `response` is assumed to be happy here, and maybe this callback still fires if the response fails for any reason? (no idea if that is the case though!)

Reply:
ah no - this will only fire if response is happy. There is a separate event in the component for unhappy path